straxen/scada.py

```diff
@@ -0,0 +1,246 @@
+import urllib
+import requests
+import pandas as pd
+import numba
+import numpy as np
+
+import strax
+import straxen
+
+try:
+    # Getting secrets:
+    SCData_URL = straxen.get_secret('SCData_URL')
+    SCLastValue_URL = straxen.get_secret('SCLastValue_URL')
+    SCADA_SECRETS = straxen.get_secret('SCADA_SECRETS')
+except ValueError:
+    # We cannot import scada (presumably because we don't have the right
+    # xenon_sectets.py file.
+    SCData_URL = SCLastValue_URL = SCADA_SECRETS = None
+
+export, __all__ = strax.exporter()
+
+
+@export
+def convert_time_zone(df, tz):
```

Comment:
Why not make 'strax' the default?

Reply:
So your argument is to better have people aware than making the choice for them, I buy that.